src/javascript/app/base/binary_loader.js

```diff
@@ -66,9 +66,9 @@ const BinaryLoader = (() => {
 
         const this_page = e.detail.getAttribute('data-page');
         if (this_page in pages_config) {
```

Comment:
I think `hasOwnProperty` is safer here, or is there any reason for looking up in the prototype chain of pages_config?